front-end/src/components/Manager/Main/EntitiesMenu/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
You must not commit the same changes to multiple branches  or pull requests. This may lead to merge conflicts, as you do not know which pull request will be merged first. Please remove duplicated changes that you already have in load-all-entities-from-server-MP branch.